source/game/gt_gladiator.cpp

```diff
@@ -562,7 +562,7 @@ static bool GT_Gladiator_SpawnEntity( StringHash classname, edict_t * ent ) {
 }
 
 Gametype GetGladiatorGametype() {
-	Gametype gt;
+	Gametype gt = {};
 
 	gt.Init = GT_Gladiator_InitGametype;
 	gt.Init2 = NULL;
```

Comment:
might as well get rid of these = NULLs and nitpicking but i normally do spaces between `{ }`